dropbox/dropbox.py

```diff
@@ -452,6 +466,39 @@ def request_json_string(self,
         else:
             raise HttpError(request_id, r.status_code, r.text)
 
+    def with_path_root(self, path_root=None):
```

Comment:
When `path_root` is `None` is a defined behavior (vs. a default behavior), I don't think it should be default argument value.

Reply:
Header omission has its own behavior, but I'm refactoring per our offline discussion.

Reply:
I think the `None` default still makes sense. I updated the comment to make this clearer, but please let me know what you think.